### ISSUE TITLE
copy / paste whole pages between two xournal instances

### ISSUE THREAD

**Issue by matthigger** [xournalpp/xournalpp#2335]
Its currently possible to copy/paste between two open xournal instances if you select a particular set of markings on the page (#2294).  **How can I copy whole pages of xournal markings (lines, images, text, highlighting ...) between two notebooks?**  I've looked around and don't believe this exists yet so I'm submitting here as a feature request.

a bit more about my use-case:
I use xournal to teach and prefer to swap between pages rather than scroll.  From semester to semester I find myself wanting to merge/split large portions of a lesson's notes and it'd be helpful to be able to grab old lectures and copy paste just those slides I'd like into a new lesson.  Currently I'm grabbing all the elements on the page with a rectangle tool and then copy / pasting but its cumbersome.

(and also, thank you for this software!  Both my wife and I teach with it daily throughout the school year and its a great resource for learning, especially now that we're teaching online)

**Comment by rolandlo** [comment]
You are right, currently it's only possible to copy selections, not whole pages. Maybe in your case it would make sense to make a copy of the whole document (in the file browser) and then (in Xournal++) delete the pages you don't need. Deleting pages is quite fast.

**Comment by matthigger** [comment]
Hi, thanks for the response.

This is a helpful workaround, though I do sometimes need to merge two distinct xournal files into a new one, drawing on slides from each.

you're right that copying the file and deleting does get most of this particular task done for me though

**Comment by Technius** [comment]
This could be nicely accomplished with an "Import from..." menu option.

**Comment by fewaltix** [comment]
Agreed, this would be a very helpful feature, even if it was just possible to copy / paste between the miniature view in the sideboard (which would be similar to the workflow in presentation or pdf software and thus quite intuitive) .
